### PR TITLE
Improve our readme

### PR DESCRIPTION
**Pull request by Firstyear**
Improves readme and docs chapter layout to make it a bit more intuitive and easier to read. 

- [ ] cargo fmt has been run
- [ ] cargo clippy has been run
- [ ] cargo test has been run and passes
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

File: kanidm_book/src/backup_restore.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+To take the backup (assuming our docker environment) you first need to stop the instance:
+
+    docker stop <container name>
+    docker run --rm -i -t -v kanidmd:/data -v kanidmd_backups:/backup \
+        kanidm/server:latest /sbin/kanidmd database backup -c /data/server.toml \
+        /backup/kanidm.backup.json
+    docker start <container name>
+
+You can then restart your instance. DO NOT modify the backup.json as it may introduce
```
Comment (yaleman) [inline kanidm_book/src/backup_restore.md:28]: They've already started the instance if they followed the steps above 😃

**Review by yaleman — CHANGES_REQUESTED** [review]
Looks great 😍 some minor tweaks as usual.

File: kanidm_book/src/backup_restore.md (hunk 1 above)
Comment (Firstyear) [reply kanidm_book/src/backup_restore.md:28]: Doesn't hurt to say it again (I think that was the original text anyway ^^; )

Comment (yaleman) [reply kanidm_book/src/backup_restore.md:28]: Yeah, just callin' it out as usual. :D All good.

**Review by yaleman — APPROVED** [review]

**Pull request merged**